### DISCUSSION TITLE
Cannot get the student pack

### DISCUSSION THREAD

**Discussion by MaryJyl** [community/community#39930] [GitHub Education]
Hi there, 
I am currently a student with Studi in France, but I have a big issue on signing in the Education Pack, please, i really need to have it in order to start my course.

**Comment by DSDmark** [comment]
Hi @MaryJyl,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Comment by MaryJyl** [comment]
I Simply want to get the full  github pack student whic permit to access a
lot of dev tools, as a student With Digital Campus Live  in France,
(remotly), This school is part of Studi Group (Galileo Global Education).
Our teacher asked and recommend to get it .

I was asked to mention the school in the map but i can't find it .
 Here is the website link: https://app.studi.fr

Le jeu. 24 nov. 2022 à 12:52, Sanskar Sahu ***@***.***> a
écrit :

**Reply by samus-aran** [reply]
Hi @MaryJyl :wave:

So you are a student of Digital Campus Live but it has no campus (hence why you can't pin point it in the map?). As you can imagine GitHub has numerous institutions across the globe but there are a few that require students & faculty to identify for us.

I would first return to your teacher and asked if they can apply (if they haven't done so already). This will help with identifying the correct school and campus even if you are remote learning. Once the school has been verified you and your fellow school peers can use it to re-apply.
